src/relay/op/type_relations.cc

```diff
@@ -87,6 +87,11 @@ Type ConcreteBroadcast(const TensorType& t1,
       oshape.push_back(s2);
     } else if (EqualConstInt(s2, 1)) {
       oshape.push_back(s1);
+    } else if (s1.as<Any>() && EqualConstInt(s2, 1)) {
```

Comment:
Just wondering, should we save 1 when broadcasting with Any?

Reply:
Any -> 1 should be Any broadcasting will promote the larger dimension here, for example:

- `1, 1 -> 1`
- `1, n -> n`